help.go

```diff
@@ -155,24 +157,91 @@ func ShowAppHelp(c *Context) (err error) {
 	return nil
 }
 
+var shortFlagRegex = regexp.MustCompile(`^-`)
```

Comment:
I would recommend to put this regex close to the source where it is used and not making it public.

Reply:
this is not making the variable public. keeping it in function will make regex compile on every invocation of method.

Reply:
Yeah I agree. Then we leave it as it currently is.